client/src/components/ProfileListing/ProfileListing.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from 'react';
+import { searchProfiles } from '../../helpers/APICalls/searchUsers';
+import { SearchProfileApiData } from '../../interface/Profile';
+import { Box, Grid } from '@mui/material';
+
+import NotFound from '../../pages/NotFound/NotFound';
+import SitterCard from './SitterCard';
+import staticData from './constants';
+
+interface Props {
+  debouncedLocation: string | null;
+  date: string;
+}
+
+const ProfileListing = ({ debouncedLocation, date }: Props): JSX.Element => {
+  const [profiles, setProfiles] = React.useState<SearchProfileApiData | null>(null);
+
+  useEffect(() => {
+    (async () => {
+      if (debouncedLocation && date) {
+        const data = await searchProfiles({ location: debouncedLocation, availability: date });
+
+        setProfiles(data);
+      }
+    })();
+  }, [debouncedLocation, date]);
+
+  const renderProfiles = () => {
+    if (debouncedLocation === 'DEMO') {
+      return staticData.map((sitter, id) => {
+        return (
+          <SitterCard
+            key={id}
```

Comment:
If this is not demo data! Would you use the sitter id as the key instead of the index

Reply:
Very true! Using indexes are not best practices when it comes to rendering a list of components. I added a `id` property to Profile so the mongoDB `id` will be used instead.